applications/workspaces/api/openapi.yaml

```diff
@@ -98,8 +98,8 @@ paths:
       tags:
         - rest                  
       summary: Used to save a Workspace to the repository. The user_id (keycloak user id) will be automatically filled with the current user
-      security:
-        - bearerAuth: []
+      # security:
```

Comment:
Was this commented for testing or on purpose?

Reply:
yes this is for testing